README.md

```diff
@@ -73,6 +73,10 @@ On other platforms, you may have to do tinkering to make things work.
 To build `workerd`, you need:
 
 * [Bazel](https://bazel.build/)
+* [Python 3](https://www.python.org/downloads/) with a `python3` executable on the `PATH`
+  * [Ensure that pip is installed as well](https://packaging.python.org/en/latest/tutorials/installing-packages/#ensure-you-can-run-pip-from-the-command-line),
+  pip is not guarenteed to be be included in the base python three installation and may be excluded
+  by your systems package manager.
```

Comment:
`s/guarenteed/guaranteed`
`s/be be/be`
`s/python three/python3`
`s/systems/system's`

Per PR #416 and other discussion, I suspect a full install of `pip3` might not be strictly necessary on Linux; installing python distutils may be sufficient.